### PR TITLE
DOC Using pure MNE Epochs

### PR DESCRIPTION
**Pull request by PierreGtch**
New tutorial for basic usage demonstrating the new features of EEGClassifier, EEGRegressor and the normalised models. Cf #457

### CODE REVIEW

**Comment by PierreGtch** [comment]
@bruAristimunha @robintibor do you think this example is clear enough to demonstrate the new capabilities of EEGClassifier?

File: docs/conf.py [hunk 1]
```diff
@@ -235,6 +237,7 @@ def linkcode_resolve(domain, info):
     'reference_url': dict(braindecode=None),
     'subsection_order': ExplicitOrder(
         [
+            '../examples/basic_training',
             '../examples/model_building',
```
Comment (bruAristimunha) [inline docs/conf.py:241]: why did you change the order @PierreGtch?

File: examples/basic_training/plot_basic_training_epochs.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+# If you want to pass parameters to your model, you can give them to the wrapper
+# with the prefix ``module__``.
+#
+
+from braindecode import EEGClassifier
+
+net = EEGClassifier(
+    'ShallowFBCSPNet',
+    module__final_conv_length='auto',
+)
```
Comment (bruAristimunha) [inline examples/basic_training/plot_basic_training_epochs.py:131]: Loved!

File: examples/basic_training/plot_basic_training_epochs.py
Comment (bruAristimunha) [inline examples/basic_training/plot_basic_training_epochs.py:1]: Super cute tutorial! Can you move the tutorial to the folder with basic decoding? I think makes more sense

Comment (bruAristimunha) [reply examples/basic_training/plot_basic_training_epochs.py:1]: moved!

**Review by bruAristimunha — APPROVED** [review]

File: examples/model_building/plot_basic_training_epochs.py [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+# To initialize it, we need to specify some parameters that we will set at random
+# values for now:
+
+model = ShallowFBCSPNet(
+    n_chans=32,
+    n_times=1000,
+    n_outputs=2,
+    final_conv_length='auto',
+)
+print(str(model))
```
Comment (bruAristimunha) [inline examples/model_building/plot_basic_training_epochs.py:75]:
```suggestion
print(model)
```

File: examples/model_building/plot_basic_training_epochs.py [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+# :class:`braindecode.EEGRegressor`. As our fake data is a classification task,
+# we will use the former.
+#
+# The wrapper :class:`braindecode.EEGClassifier` expects a model class as its first argument but
+# to facilitate the usage, you can also simply pass the name of any braindecode model as a string.
+# The wrapper will automatically find and instantiate the model for you.
+# If you want to pass parameters to your model, you can give them to the wrapper
+# with the prefix ``module__``.
+#
+
```
Comment (bruAristimunha) [inline examples/model_building/plot_basic_training_epochs.py:125]:
```suggestion
from skorch.dataset import ValidSplit
```

**Review by bruAristimunha — COMMENTED** [review]
I was looking the render webpage, and I think we need one small adjust.

**Comment by bruAristimunha** [comment]
Thank you @PierreGtch
